test.py

```diff
@@ -38,7 +38,20 @@ def test_kafka_message_send(self):
         Test that no errors are thrown, and it doesn't hang when you send a message
         """
         producer = HerokuKafkaProducer(**kafka_params)
-        producer.send(TOPIC1, b"some message")
+        #producer.send(TOPIC1, b"some message")
```

Comment:
I think this comment is not required anymore

Reply:
lol xD good spot ahahha